.gitignore

```diff
@@ -7,6 +7,7 @@ __pycache__
 *.sw[a-p]
 ._*
 .DS_Store
+nohup.out
```

Comment:
Why is this file added to `.gitignore`?

Reply:
`nohup.out` is a file usually produced when one runs a script or an executable using `nohup`.  `nohup` allows running a job in the background and closing the terminal will allow the job to continue running.
This is especially useful for scripts such as `build_all.sh` that take a while.  
It can be viewed as a file similar to a "log".

Reply:
Thank you @aerorahul .   I am familiar with `nohup.out`.   I often run the GDASApp build in the background with `nohup`.   I was just wondering why we put it in the `.gitignore` list.   I like to see `nohup.out` in the `git status` listing.   When I commit, I manually `git add` each file I intend to commit.   Was `nohup.out` added to `.gitignore` to avoid accidentally committing this file to the repo?

Reply:
I agree with @RussTreadon-NOAA that I like seeing files like this in my "git status".

Reply:
@RussTreadon-NOAA  @JessicaMeixner-NOAA 
`nohup.out` was added to `.gitignore` to avoid accidentally committing this file to the repo similar to the other files listed above; such as executables, logs, compiled artifacts.
Users should be able to do an `ls` and still see this file.

Reply:
The file could always be removed later if accidentally committed and be avoided by manually adding each commit field.

Reply:
Everywhere I have seen, this file is part of `.gitignore` to precisely preventing manual removing an accidental addition of this (and other types of files).

@RussTreadon-NOAA Is your comment suggesting we don't add this to `.gitignore`?  Am I missing something? 

Either way, this should not be a blocker for this PR.

Reply:
@aerorahul .   Yes, I don't think `nohup.out` should be in `.gitignore`.  If it's standard practice to place this file in `.gitignore`, I'm OK with this.   I agree.  This is not a blocker for this PR.

Reply:
Ok. we will review this user request against best practice and hold this for another time.